crates/modelardb_server/src/metadata/mod.rs

```diff
@@ -585,7 +554,7 @@ impl MetadataManager {
         end_time: Option<Timestamp>,
         min_value: Option<Value>,
         max_value: Option<Value>,
-    ) -> Result<Vec<Uuid>> {
+    ) -> Result<Vec<CompressedFile>> {
```

Comment:
I initially thought the extra information in `CompressedFile` was needed to transfer metadata to the manager later but after more consideration it might not be needed, so I have made the suggested change.